test/go_modules/test_repo/third_party/go/BUILD_FILE

```diff
@@ -98,3 +98,14 @@ go_module(
     ],
     licences = ["Apache-2.0"]
 )
+
+# Tiny CGO module that tests relative C header files imports.
+filegroup(
+    name = "cgo_example_srcs",
+    srcs = ["cgo_example"],
+)
+go_module(
+    name = "cgo",
+    module = "cgo_example",
+    download = ":cgo_example_srcs",
+)
```

Comment:
It's a project with quite a bit of transitive deps, we would need the go-get tool for that which would make this test BUILD file more difficult to reason about. This simple test should be enough since it replicates the scenario where delve was failing to build before.